projects/plugins/jetpack/changelog/fix-forms-select-field-icon

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Forms: Fix dropdown icon style
```

Comment:
Do I need this second changelog item? First is for the forms package, this is for the Jetpack plugin.

Reply:
I think you'll end up getting Jetpack changelog in addition to package changelog if you answer "yes" to "should site owners know about this change?"

You can delete it, but really no biggie to just have it too.